vm/src/air_public_input.rs

```diff
@@ -56,14 +56,14 @@ impl From<(usize, usize)> for MemorySegmentAddresses {
 
 #[derive(Serialize, Debug)]
 pub struct PublicInput<'a> {
-    layout: &'a str,
-    rc_min: isize,
-    rc_max: isize,
-    n_steps: usize,
-    memory_segments: HashMap<&'a str, MemorySegmentAddresses>,
-    public_memory: Vec<PublicMemoryEntry>,
+    pub layout: &'a str,
+    pub rc_min: isize,
+    pub rc_max: isize,
+    pub n_steps: usize,
+    pub memory_segments: HashMap<&'a str, MemorySegmentAddresses>,
+    pub public_memory: Vec<PublicMemoryEntry>,
     #[serde(rename = "dynamic_params")]
-    layout_params: Option<&'a CairoLayout>,
+    pub layout_params: Option<&'a CairoLayout>,
```

Comment:
from the prover side, we don't really need the layout params as public, at least for the moment. With the name which appears in the `layout` field should be ok 👍

Reply:
great!
Done [509bed0](https://github.com/lambdaclass/cairo-vm/pull/1474/commits/509bed067ef718e2fa622b71f68351446f17ee5a)